doc/api.md

```diff
@@ -120,6 +122,10 @@ presented edition and [warnings](#warnings).
 - `locale` *(optional, default: "en")*
   - Accepts: An available locale from the [Rails I18n gem][i18n-gem]
   - Specifies the locale of the edition.
+- `major_published_at` *(optional)*
+  - Specifies when a major update was published.
+- `minor_published_at` *(optional)*
```

Comment:
I don't think we need to add these in yet - these would also require a schema change. I think it's fine for this step in the process to fill these in based on `public_updated_at` 

Good to add `first_published_at` in though since that should have been there before